src/server/journal/journal_slice.cc

```diff
@@ -159,5 +164,22 @@ void JournalSlice::UnregisterOnChange(uint32_t id) {
   change_cb_arr_.erase(it);
 }
 
+void JournalSlice::PingsLoop(std::chrono::milliseconds period_ms) {
+  std::string command = "PING";
+  MutableSlice slice{command};
+  CmdArgList args{&slice, 1};
+  Entry::Payload payload = std::make_pair(command, args);
+  Entry entry(/*txid=*/0, Op::COMMAND, /*dbid=*/0,
```

Comment:
To me it looks like a good case to add another opcode Op::PING
call
AddLogRecord(Entry{0, journal::Op::PING, 0, 0, {}, true})

Reply:
I abuse the fact that I actually send a command because then it goes through the command dispatcher and I can use it to force trigger the ACK.

I can implement it with a different opcode but it's easier this way :laughing: